.simplecov

```diff
@@ -0,0 +1,13 @@
+SimpleCov.start "rails" do
+  add_filter "config/"
+  add_filter "spec/"
+  add_filter "services/migration_helpers/"
+  add_filter "lib/tasks"
```

Comment:
I added this filter because they weren't covered. I'm not sure why it wasn't picked up beforehand.

Reply:
we do have some specs for `lib/tasks/helpers/` which are getting excluded by this but not sure why it was not complaining about other unspeced  `lib/tasks/helpers/` though so 🤷‍♂️

Reply:
Ah ok, let's remove this filter then and add tests for what is missing!